tests/unit/test_likelihoods.py

```diff
@@ -75,12 +75,6 @@ def test_base_likelihood_init(self, problem):
         assert likelihood._n_parameters == 1
         assert np.array_equal(likelihood._target, problem._target)
 
-    @pytest.mark.unit
```

Comment:
I think we still want this test? Looking at the above class, `BaseLikelihood` still returns NotImplemented from __call__(). It would be good to keep a test on that.